src/app/ui/multiParamManager.js

```diff
@@ -207,7 +207,10 @@ class MultiParamManager {
     if (this.funABI.payable === true) {
```

Comment:
Does this line need to check if mutable function returns payable and not check this.funABI.payable?

Reply:
sure, 10x